README.md

```diff
@@ -62,27 +60,22 @@ Offical exam objectives you review and understand in order to pass the test.
   #### Extra helpful material
   
   - [x] [Slack](#slack)
-  - [x] [Books](#books)
-  - [x] [Youtube Videos](#youtube-videos)
-  - [x] [Webinars](#webinars)
-  - [x] [Containers and Kubernetes Security Training](#containers-and-kubernetes-security-training)
-  - [x] [Extra Kubernetes security resources](generic-kubernetes-containers-security/Kubernetes.md)
+  - [x] [Twitch](#twitch)
+  - [x] [Books](generic-kubernetes-containers-security/Kubernetes.md#books)
+  - [x] [Youtube Videos](generic-kubernetes-containers-security/Kubernetes.md#youtube-videos)
+  - [x] [Containers and Kubernetes Security Training](generic-kubernetes-containers-security/Kubernetes.md#containers-and-kubernetes-security-training)
```

Comment:
training dropped?

Reply:
I think it should be new changes, as the previous one I return it back to main README